Source/options.cpp

```diff
@@ -259,6 +259,8 @@ void LoadOptions()
 	sgOptions.Gameplay.bRandomizeQuests = GetIniBool("Game", "Randomize Quests", true);
 	sgOptions.Gameplay.bShowMonsterType = GetIniBool("Game", "Show Monster Type", false);
 	sgOptions.Gameplay.bDisableCripplingShrines = GetIniBool("Game", "Disable Crippling Shrines", false);
+	sgOptions.Gameplay.nSPanelHAlign = GetIniInt("Game", "Sub Panel Horizontal Alignment", 0);
+	sgOptions.Gameplay.nSPanelVAlign = GetIniInt("Game", "Sub Panel Vertical Alignment", 0);

```

Comment:
I think s stands for side panels, not sub panels

Reply:
Modified